src/collector/LogCollectorState.js

```diff
@@ -51,6 +51,10 @@ module.exports = class LogCollectorState {
       return;
     }
 
+    if (this.config.processLog) {
+      entry[1] = this.config.processLog(entry);
```

Comment:
let's allow processing the whole entry, so `entry = this.config.processLog(entry);`